docs/scripts_md/cleanup_paths_of_violation_tables.md

```diff
@@ -0,0 +1,26 @@
+### update\_MincPath\_field($dbh, $table, $id\_field, $file\_field)
+
+Greps all the files present in a given table and updates its location to the file
+present in the trashbin subdirectory of the LORIS-MRI data directory.
```

Comment:
This description is not clear. I kind of understand using the description of this PR but not really sure. Maybe need to rephrase.